envi/archs/arm/const.py

```diff
@@ -283,11 +283,18 @@
 IENC_MEDIA_USADA8   = ((IENC_MEDIA << 8) + 6) << 8
 IENC_MEDIA_EXTEND   = ((IENC_MEDIA << 8) + 7) << 8
 IENC_MEDIA_PACK     = ((IENC_MEDIA << 8) + 8) << 8
+IENC_MEDIA_SBFX     = IENC_MEDIA_PACK #FIXME
```

Comment:
ok, so it looks like this is the PR needing most of the initial work.  
please rework the naming. 
IENC_ has to do with the decoding subsystem
INS_ should be the name of the instruction.